app/budget/services/budgetService.js

```diff
@@ -117,6 +117,45 @@ budgetApp.factory("budgetService", this.budgetService = function($http, $q, $win
 
 			return deferred.promise;
 		},
+		updateInstructorType: function(instructorType) {
+			var deferred = $q.defer();
+
+			$http.put(serverRoot + "/api/budgetView/instructorTypes/" + instructorType.id, instructorType, { withCredentials: true })
```

Comment:
Seems like we use this pattern often enough there should be a helper function that performs the request, defines success and error, and returns a promise that could turn a lot of these into one-liners.